edx_solutions_organizations/tests.py

```diff
@@ -501,7 +504,10 @@ def test_organizations_users_get_with_grades(self):
             data = {"id": user.id}
             response = self.do_post(users_uri, data)
             self.assertEqual(response.status_code, 201)
-        response = self.do_get('{}?{}&course_id={}'.format(users_uri, 'include_grades=True', self.course.id))
+
+        course_id = {'course_id': '{}'.format(unicode(self.course.id))}
```

Comment:
this should be renamed to `params` instead of `course_id` and don't need to format like this
```
params = {'course_id': unicode(self.course.id), 'include_grades':True}
response = self.do_get(users_uri, query_parameters=params)
```